waterlog/src/Components/LeakageHistory.js

```diff
@@ -0,0 +1,18 @@
+import React, {
+    Component
+} from 'react';
+
+class LeakageHistoryComponent extends Component {
+
+    render() {
+        return ( 
+        <div >
```

Comment:
The <div> element helps you to divide or organize your elements.
Wrap your texts with either a paragraph, header or span so that you select it independently of the div when the need arises

Reply:
I have adjusted this.

Reply:
updated